types/systemjs/index.d.ts

```diff
@@ -318,7 +322,6 @@ declare namespace SystemJSLoader {
          * Useful when writing a custom instantiate hook or using System.set.
          */
         newModule(object: any): any;
```

Comment:
Same, add missing generic value.

Reply:
All uses of generics were raising a dtslint error. Since the generics are not associated with either the parameters or the return values of the declaring functions, they essentially behave like a type assertion.

The rule is explained here. https://github.com/Microsoft/dtslint/blob/master/docs/no-unnecessary-generics.md